pkg/kubelet/rkt/rkt.go

```diff
@@ -1320,6 +1393,13 @@ func (r *Runtime) KillPod(pod *api.Pod, runningPod kubecontainer.Pod) error {
 		return err
 	}
 
+	if !kubecontainer.IsHostNetworkPod(pod) {
+		if err := r.cleanupPodNetwork(&runningPod, true); err != nil {
```

Comment:
@dcbw I guess the docker pod works fine today because the `pause` container hardly dies, so in most case the only time it dies is when it's been explicitly killed, which invokes the `networkplugin.TearDown()` accordingly.